docs/concepts/redirects.mdx

```diff
@@ -40,105 +38,148 @@ will fail, however, for https://anotherapp.com will succeed on any path.
 Ory Cloud has a total of six flows, **Login**, **Registration**,
 **Verification**, **Recovery**, **Settings** and **Logout** which can be
 configured to redirect back to any URL. A common use case would be to redirect
-the user to your application home screen after a logout or to a specific url on
-a sub-domain after a settings password update. On project creation the redirect
-flows are set to the default Ory Cloud Managed UI. For more information on the
-Ory Managed UI, please see the [Managed User Interface](./managed-ui.mdx)
-documentation.
-
-### Login, Registration and Settings
-
-The Login and Registration flows have three fields (all optional):
-
-1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-OIDC Redirect URL` (**not supported**)
-
-And Settings has three fields (all optional):
+the user to your application home screen after a logout or to a specific URL on
+a sub-domain after a settings password update.
 
-1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-Profile Redirect URL`
+On project creation the redirect flows are set to the default
+[Managed User Interface](./managed-ui.mdx).
 
-:::info
+### Login, Registration, and Settings
 
-Setting `Post-OIDC` redirects for Login and Registration will be supported in a
-future release. For more information see the
-[Ory Cloud Roadmap](../cloud/cloud-roadmap.mdx).
-
-:::
-
-When setting the `Default Redirect URL` as an FQDN (fully qualified domain name)
-such as `https://domain.example/login` it will overwrite the
-`Global Redirect URL`. As a path it will replace the `Global Redirect URL` path,
-for example a path such as `/login` will alter
-`https://domain.example/global/path` to `https://domain.example/login`.
+The Login and Registration flows have three fields (all optional):
 
-On the sub-conditions such as `Post-Password` or `Post-OIDC` an FQDN or a path
-can be set. In the case of an FQDN it will replace the closest relative starting
-with the Login/Registration `Default Redirect URL` and then the
-`Global Redirect URL`. With a path it will use the closest relative as the base
-URL as described above with the `Default Redirect URL`.
+1. Default Redirect URL domain or path relative to your application URL
+2. Post-Password Redirect URL
+3. Post-OIDC Redirect URL
+
+```yaml file="kratos.config.yml"
+selfservice:
```

Comment:
As a follow up we should show how to set this using tabs:

1. Self-Hosted Kratos (this config file here)
2. Ory Console (screenshot)
3. Ory CLI (`ory patch project .. --replace /.../...`)

I'm ok not doing this in this PR but can you please create a follow up issue and tackle this soon? Because this section is otherwise not very helpful for cloud customers. Please link the issue here